.github/workflows/ci.yml

```diff
@@ -0,0 +1,120 @@
+name: Alfresco Transform Core CI
+
+on:
+  push:
+    branches:
+      - feature/**
+      - fix/**
+      - ATS-**
+      - ACS-**
+      - MNT-**
+  pull_request:
+    branches:
+      - master
+      - feature/**
+      - fix/**
+      - SP/**
+      - HF/**
+      - ATS-**
+      - ACS-**
+      - MNT-**
+  workflow_call:
+  workflow_dispatch:
+
+env:
+  DOCKERHUB_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+  DOCKERHUB_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+  GITHUB_ACTIONS_DEPLOY_TIMEOUT: 60
+  NEXUS_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+  NEXUS_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+  QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
+  QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
+
+jobs:
+  veracode:
+    name: "Veracode - Source Clear Scan (SCA)"
+    runs-on: ubuntu-latest
+    if: >
+      github.ref_name == 'master' || github.event_name == 'pull_request'
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.32.0
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.32.0
+      - name: "Before install"
```

Comment:
Replace `"Before install"` with something that better described the intention of the step.